docs/accessories/various_questions.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Common Asked Questions
+project: accessories
+---
+
+## General
+
+#### Can I adjust where something is equipped to another slot?
+
+Any item can be equipped within a given slot but it does not mean such item will function as it would if not specifically designed for such. Accessories API only attempts to add the ability to store and retrieve items and due to minecraft limitations it is not possible to just extend the current system and hook into such.
+
+#### Is it possible to remove a slot altogether?
+
+Such can be done but it is not recommend for Modders to take such drastic action unless really needing to do such since It may mess with other peoples mods ability to equip accessories. 
+
+Datapackers should also be cautious for similar reasons but for Modpack developers its understandable if they desire to rebalance their pack to either condense slots or balance the amount of slots but do note that **some developers will lock** their Accessories to specific slots either by looking only in those slots or controlling functionality to work only in the given slot
+
+#### Does enchantments work with equipped accessories?
+
+If the item is able to be enchanted with a given enchantment then it is possible for the such enchantment to work almost natively with the base equipment system found within Minecraft but some edge case may exists where the enchantment will not function properly if outside the scope of the API.
```

Comment:
Its mainly that the enchantment may be only checked for in minecrafts equipment API instead of being more globally executed by Enchantment Helper Class